ui/app/home/home-ctrl.js

```diff
@@ -1,11 +1,238 @@
-(function () {
+(function() {
   'use strict';
   angular.module('demoCat.home', []);
   angular.module('demoCat.home')
-    .controller('HomeCtrl', ['$scope', function ($scope) {
-      var model = {};
-      angular.extend($scope,{
-        model: model
-      });
-    }]);
+    .factory('HomeModel', [
+      function() {
+        return {
+          user: null,
+          vanguard: null,
+          donttouch: null,
+          spotlight: null
```

Comment:
This needs to be:
spotlight: []

As-is it doesn't work for me when the spotlight list is empty.